modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -61,6 +57,34 @@ definition {
 		}
 	}
 
+	@description = "LPS-172949 - Finance user should edit Organizations with 'Awaiting Finance Review' status"
+	@priority = 4
+	test FinanceUserCanEditOrganization {
```

Comment:
Hey @joelrocha68, the test name must begin with an action. Could you change it please?
Sugestion: `CanFinanceUserEditOrganization`